boards/arm/pan1783_pan1783a_pan1783apa_evb/doc/index.rst

```diff
@@ -0,0 +1,72 @@
+.. _pan1783_evb:
+
```

Comment:
Seemingly the .jpg files seem to have grown in size, can you go back to the old images and just rename them instead of changing them?

Reply:
Yes, I noticed, that the old ones have a different resolution. So I reworked all the images to the same resolution (width), which looks much better. What I could do is decrease the resolution again. What size should I not exceed for the images?

Reply:
Aiming for <50KB would be great. Please consider using webp format (which would also support transparency, assuming you have access to the "original" pictures).

Reply:
I tried to find a good compromise between resolution and quality and changed them into webp format.

Reply:
Thanks! Nice to have transparent background too! Given the relatively resolution, you can actually aim for much smaller size without noticeable degradation in quality. `cwebp pan1783a_evb.webp -q 70 -o test.webp` gives me a 24K file. You may want to work with your original lossless file as the input of cwebp to get even better quality.

Sorry if that sounds nitpicky but given that the images live in the Git repo, adding only ~70K instead of 150K of binary files would actullay make quite a difference. In fact, given there's hardly any visible difference between the 3 PCB, may I suggest you only include *one* image as an illustration? Thanks!

Reply:
Okay, I changed the description. So having just one image is ok for me now. With my original *.png as the source, I came down to a size of 41K for the webp image.